docs/src/reference/the-traversal.asciidoc

```diff
@@ -1434,6 +1436,15 @@ the key,value pairs for those vertices.
 <6> Find all age-properties and emit their value.
 <7> Find all vertices that do not have an age-property and emit their name.
 <8> Find all "person" vertices that have a name property that starts with the letter "m".
+<9> Property key values cannot be `null` and therefore produces no matches.
```

Comment:
Not that important, but _produce_ without an _s_, right?

Reply:
uhh - depending on how you read the subject of the sentence it could go either way. i intended "produces" so i'll reword.